tools/ci_build/github/azure-pipelines/nuget/templates/gpu.yml

```diff
@@ -30,23 +30,26 @@ jobs:
      mkdir $(Build.ArtifactStagingDirectory)\testdata
      copy $(Build.BinariesDirectory)\RelWithDebInfo\RelWithDebInfo\custom_op_library.* $(Build.ArtifactStagingDirectory)\testdata
 
-- template: ../../templates/win-ci.yml
+- template: ../../templates/win-ci-2019.yml
   parameters:
-    AgentPool : $(AgentPoolWin)
+    AgentPool : 'Win-GPU-2019'
+    ArtifactName: drop-nuget-dml
     JobName: 'Windows_CI_GPU_DML_Dev'
-    BuildCommand: '$(Build.SourcesDirectory)\tools\ci_build\build.py --build_dir $(Build.BinariesDirectory) --skip_submodule_sync --cmake_path $(Build.BinariesDirectory)\cmake\bin\cmake.exe --ctest_path $(Build.BinariesDirectory)\cmake\bin\ctest.exe  --enable_pybind --build_shared_lib  --build_csharp --enable_onnx_tests --use_winml --use_dml'
+    BuildCommand: --build_dir $(Build.BinariesDirectory) --skip_submodule_sync --build_shared_lib --use_featurizers --enable_onnx_tests --use_telemetry  --use_winml --use_dml --cmake_generator "Visual Studio 16 2019"
```

Comment:
>use_telemetry [](start = 138, length = 13)

Cool, I see the TELEMETRYGUID was added in Nuget GPU Pipeline (OnnxRuntime-Github)

Reply:
Are `cmake_path` and `ctest_path` missing on purpose?